kuka_external_control_sdk/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>kuka_external_control_sdk</name>
+  <version>1.2.1</version>
+  <description>Client library for KUKA external control interfaces</description>
```

Comment:
Maybe client libraries for external interfaces supported by KUKA? This is very minor, but I think we can emphasize that this is not a singular client library.

Reply:
done